Automattic/wp-calypso
# Image Editor: hasChanges should only be true if a user action has occured

Alternative approach to #9060 that uses 2 new actions to differ between handling user actions and image editor init work.

## Testing Instructions
- http://calypso.localhost:3000/devdocs/blocks/image-editor
- "Reset" button should be disabled on page load.
- Perform any action in the editor ( aspect selection, crop, flip etc)
- "Reset" button should be enabled
- Click "Reset". Image should return to original.
